console-subscriber/src/lib.rs

```diff
@@ -641,13 +706,58 @@ impl fmt::Debug for ConsoleLayer {
 
 impl Server {
     // XXX(eliza): why is `SocketAddr::new` not `const`???
+    /// A [`Server`] by default binds socket address 127.0.0.1 to service remote
+    /// procedure calls.
+    ///
+    /// Note that methods like [`init`][`crate::init`] and
+    /// [`spawn`][`crate::spawn`] will parse the socket address from the
+    /// `TOKIO_CONSOLE_BIND` [environment variable] before falling back on
+    /// constructing a socket address from this default.
+    ///
+    /// See also [`Builder::server_addr`].
+    ///
+    /// [environment variable]: `Builder::with_default_env`
     pub const DEFAULT_IP: IpAddr = IpAddr::V4(Ipv4Addr::new(127, 0, 0, 1));
+
+    /// A [`Server`] by default binds port 6669 to service remote procedure
+    /// calls.
+    ///
+    /// Note that methods like [`init`][`crate::init`] and
+    /// [`spawn`][`crate::spawn`] will parse the socket address from the
+    /// `TOKIO_CONSOLE_BIND` [environment variable] before falling back on
+    /// constructing a socket address from this default.
+    ///
+    /// See also [`Builder::server_addr`].
+    ///
+    /// [environment variable]: `Builder::with_default_env`
     pub const DEFAULT_PORT: u16 = 6669;
 
+    /// Starts the gRPC service with the default gRPC settings.
+    ///
+    /// To configure gRPC server settings before starting the server, use
+    /// [`serve_with`] instead. This method is equivalent to calling [`serve_with`]
+    /// and providing the default gRPC server settings:
+    ///
+    /// ```rust
+    /// # async fn docs() -> Result<(), Box<dyn std::error::Error + Send + Sync + 'static>> {
+    /// # let (_, server) = console_subscriber::ConsoleLayer::new();
+    /// server.serve_with(tonic::transport::Server::default()).await
+    /// # }
+    /// ```
```

Comment:
i guess rustdoc can't resolve this link? https://app.netlify.com/sites/tokio-console/deploys/61ba771eb07ff90007653aa9
```suggestion
    /// ```
    /// [`serve_with`]: Builder::serve_with
```

Reply:
Do need to put quotes around `Builder::serve_with`, like:
```
/// [`serve_with`]: `Builder::serve_with`
```
?

Reply:
@pnkfelix not for doc links. That should be enough.